src/components/Table/styles.tsx

```diff
@@ -0,0 +1,175 @@
+  border-collapse: collapse;
+  table-layout: fixed;
+  display: table;
+  border-spacing: 0px;
+  width: 100%;
+
+  ${props =>
+    props.alignments &&
+    props.alignments.map(
+      (alignment, index: number) =>
+        alignment &&
+        css`
+          td:nth-child(${index + 1}) {
+            text-align: ${alignment};
+          }
+        `,
+    )};
+
+  ${() => media.mobile`
+    min-width: 100%;
+    width: initial;
+    `};
+
+  ${() => media.tablet`
+    min-width: 100%;
+    width: initial;
+    `};
+`;
+
+export const TableHead = styled.thead`
+  display: table-header-group;
+`;
+
+export const TableBody = styled.tbody`
+  display: table-row-group;
+`;
+
+export const TableRow = styled.tr<RowProps>`
+  height: ${props => props.theme.table.height.row};
+  background: ${props => (!props.isActive ? props.theme.table.background.main : props.theme.table.background.active)};
+  cursor: ${props => (props.onClick ? 'pointer' : 'default')};
+
+  ${props =>
+    !props.isStatic &&
+    css<RowProps>`
+      &:hover {
+        background: ${props =>
+          !props.isActive ? props.theme.table.background.hover : props.theme.table.background.active};
+      }
+    `}
+
+  &:hover {
+    td div.actions {
+      display: flex;
+    }
+  }
+`;
+
+export const SubRowTableRow = styled(TableRow)`
+  + tr > td {
+    border-top: ${props => props.theme.table.border.main};
+  }
+
+  > td {
+    padding-top: 0;
+  }
+`;
+
+const tableCellProps = css<CellProps>`
+  padding: ${props => props.theme.table.padding.main};
+  border-bottom: ${props => props.theme.table.border.main};
+
+  &:last-child {
+    padding-right: ${props => props.theme.table.padding.end};
+  }
+
+  ${props =>
+    props.align
+      ? css`
+          &&& {
+            text-align: ${props.align || 'left'};
+          }
+        `
+      : 'text-align: left;'}
+
+  ${props =>
+    props.size &&
+    css`
+      min-width: ${props.size}px;
+      width: ${props.size}px;
+    `}
+`;
+
+export const TableHeadCell = styled.th`
+  color: ${props => props.theme.table.color.header};
+  font-size: ${props => props.theme.table.fontSize.main};
+  text-transform: uppercase;
+  font-weight: 500;
+
+  ${tableCellProps};
+`;
+
+type HeadCellWithIconProps = {
+  hiddenIcon?: boolean;
+  active?: boolean;
+};
+
+export const TableHeadCellWithIcon = styled(TableHeadCell)<HeadCellWithIconProps>`
+  > span {
+    display: flex;
+    align-items: center;
+    cursor: ${props => (props.active ? 'pointer' : 'default')};
+    transition: color 0.2s ease-in;
+    justify-content: ${props =>
+      props.align === 'right' ? 'flex-end' : props.align === 'center' ? 'center' : 'flex-start'};
+
+    &:hover {
+      color: ${props => (props.active ? props.theme.table.color.hover : props.theme.table.color.header)};
+
+      svg {
+        fill: ${props => (props.hiddenIcon ? props.theme.table.color.hoverInactive : props.theme.table.color.hover)};
+      }
+    }
+  }
+
+  svg {
```

Comment:
This selector could mess with svg's being used in the header besides the sort icon

Reply:
I'll add a className to it